benches/src/lib.rs

```diff
@@ -95,7 +92,7 @@ pub struct VmBench {
     pub prepare_call: Option<PrepareCall>,
     pub dummy_contract: Option<ContractId>,
     pub contract_code: Option<ContractCode>,
-    pub prepare_db: Option<Box<dyn FnMut(VmDatabase) -> io::Result<VmDatabase>>>,
+    pub prepare_db: Option<Box<dyn FnMut(VmDatabase) -> anyhow::Result<VmDatabase>>>,
```

Comment:
Did you end up deciding to use `anyhow` after all? I wasn't part of the conversation, but just for thoroughness's sake, I disagree with using `anyhow` in a lib.

Reply:
For example, did you consider something like this:
https://crates.io/crates/thiserror-no-std

Reply:
At least for this PR, I'm not aiming to take a stand on the use of `anyhow` (pro or con) - I think that issue deserves its own conversation and follow up. For now, I'm simply removing the use of `std`, which includes `io` errors/results. For now, I am replacing these instances with `anyhow`, and we can later explore how best to do error handling as a practice. `thiserror-no-std` warrants a look though!

Reply:
Okay. Understood. I'm worried that it won't get prioritized, but I don't want to hold up this work because of that.